src/components/section/section.jsx

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styles from './section.css';
+
+export default function Section(props) {
+  return (
+    <section className={props.className || styles.section}>
+      <h2 className={styles.subheading}>{props.title}</h2>
+      {props.description && <p className={styles.description}>{props.description}</p>}
+      {props.children}
+    </section>
+  );
+}
+
+Section.propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  description: PropTypes.string,
+  title: PropTypes.string.isRequired,
+};
+
+Section.defaultProps = {
+  children: [],
```

Comment:
I haven't seen this pattern before. Normally, children default to `undefined` since it is kinda weird to assume either a list of child element or a single child element (which is not an array):

- 0 children is `[]`
- 1 child is `element`
- Many children is `[...]`

Reply:
Oh man, that's frustrating. I assumed it would be `[element]`. I'll change to `undefined`.